File: jyall-android-common/src/main/java/com/jyall/android/common/evnetbus/EventBusCenter.java
package com.jyall.android.common.evnetbus;

/**
 */
public class EventBusCenter<T> {

    /**
     * 接收的数据
     */
    private T data;
    /**
     * 接收码
     */
    private int eventCode = -1;

    public EventBusCenter(int eventCode) {
        this.eventCode = eventCode;
    }

    public EventBusCenter(int eventCode, T data) {
        this.eventCode = eventCode;
        this.data = data;
    }


    public int getEvenCode() {
        return eventCode;
    }

    public T getData() {
        return data;
    }


}

File: app/src/main/java/com/sitong/changqin/base/EventBusCenter.kt
package com.jyall.bbzf.base

/**
 */
class EventBusCenter<T> {

    /**
     * 接收的数据
     */
    var data: T? = null
    /**
     * 接收码
     */
    var evenCode = -1
        private set

    constructor(eventCode: Int) {
        this.evenCode = eventCode
    }

    constructor(eventCode: Int, data: T) {
        this.evenCode = eventCode
        this.data = data
    }


}
